src/components/Replies/GetReply/AllReplies/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FunctionComponent, useState } from "react";
+import { Box, Button, Grid, makeStyles, Typography } from "@material-ui/core";
+import TabViewIntroSection from "../../../TabViewIntroSection";
+
+const getAllReplyEndpoint = ["GET /comments/:commentId/replies"];
+const getAllReplyHeading = "GET all replies";
+const getAllReplySubtitle = "Need to get some replies?";
+
+const filters = [
+  "By flag state",
+  "By reference ID",
+  "By owner ID",
+  "By origin",
```

Comment:
Remember that this [endpoint ](https://comment.microapi.dev/#/Comments%20and%20replies/get_comments__commentId__replies)has 2 filters.
```suggestion
  "By flag state",
  "By owner ID",
```